etc/uams/uams_dhx_passwd.c

```diff
@@ -274,8 +285,13 @@ static int passwd_logincont(void *obj, struct passwd **uam_pwd,
 
     /* check for session id */
     memcpy(&sessid, ibuf, sizeof(sessid));
-    if (sessid != dhxhash(obj))
+    if (sessid != dhxhash(obj)) {
+    /* Log Entry */
+           LOG(log_info, logtype_uams, "uams_dhx_passwd.c :passwd Session ID - DHXHash Mismatch -- %s",
```

Comment:
Isn’t “error” log level more appropriate for this message? I think users would want to have some context to why their auth is failing. Thoughts?

Reply:
Agreed that there should be parity across the two modules. On second thought, let’s keep it as is.